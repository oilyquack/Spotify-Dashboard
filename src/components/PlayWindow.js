import React from "react";

function PlayWindow({ artist, album, coverArtSrc, trackName }) {
  return (
    <div className="app__playWindow">
      <h1>{artist}</h1>
      <img className="app__playWindow__img" src={coverArtSrc} />
      <p>{trackName}</p>
      <p>
        <i>{album}</i>
      </p>
    </div>
  );
}

export default PlayWindow;
