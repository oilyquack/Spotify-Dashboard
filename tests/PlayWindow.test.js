import React from "react";
import PlayWindow from "../src/components/PlayWindow";
import renderer from "react-test-renderer";

describe("PlayWindow", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<PlayWindow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
