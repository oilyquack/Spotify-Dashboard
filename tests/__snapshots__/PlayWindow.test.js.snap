// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayWindow matches the snapshot 1`] = `
<div
  className="app__playWindow"
>
  <h1 />
  <img
    className="app__playWindow__img"
    src={undefined}
  />
  <p />
  <p>
    <i />
  </p>
</div>
`;
